bip-0142.mediawiki

```diff
@@ -134,7 +134,7 @@ When the same public key is encoded as P2WPKH, the scriptPubKey becomes:
   
     OP_0 <010966776006953D5567439E5E39F86A0D273BEE>
 
-Using 0x06 as witness version, followed 0x00 as witness version, and a 0x00 padding, the equivalent P2WPKH address is:
+Using 0x06 as witness version, followed by 0x00 as witness version, and a 0x00 padding, the equivalent P2WPKH address is:
```

Comment:
You might want to include https://github.com/bitcoin/bips/pull/290 here.
It should be `Using 0x06 as address version, followed by 0x00 as witness program version[...]`